.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Code owners
 
 # Reviewers for any change in the repository
-* @paolomorandini @mjquinn-google @bdw-g @shevek-google
+* @paolomorandini @mjquinn-google @arotenberg-google @shevek-google
```

Comment:
Needs @arotenberg-google to accept collaboration invite (granting write permissions) that was recently issued